### PR TITLE
store the y_true and y_pred now to make the plots asked by reviewer

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: compute_benchmark_age_prediction.py [hunk 1]
```diff
@@ -153,7 +156,7 @@ def load_benchmark_data(dataset, benchmark, condition=None):
 
     if benchmark == 'filterbank-riemann':
         frequency_bands = bench_cfg['frequency_bands']
-        features = mne.externals.h5io.read_hdf5(
```
Comment (agramfort) [inline compute_benchmark_age_prediction.py:156]: mne.externals is now gone

File: compute_benchmark_age_prediction.py [hunk 2]
```diff
@@ -260,7 +264,14 @@ def run_benchmark_cv(benchmark, dataset):
             f"'{benchmark}' on dataset '{dataset}'")
         return
 
-    metrics = [mean_absolute_error, r2_score]
+    ys_true, ys_pred = [], []
+
+    def mean_absolute_error_with_memory(y_true, y_pred):
+        ys_true.append(y_true)
+        ys_pred.append(y_pred)
+        return mean_absolute_error(y_true, y_pred)
```
Comment (agramfort) [inline compute_benchmark_age_prediction.py:272]: @dengemann this is the trick I used to avoid running cross_val_predict

Comment (dengemann) [reply compute_benchmark_age_prediction.py:272]: Smart! I love it!!!

**Comment by dengemann** [comment]
Thanks a lot @agramfort !!

**Pull request merged**